src/main/java/com/machine/learning/classifier/Classifier.java

```diff
@@ -0,0 +1,21 @@
+package com.machine.learning.classifier;
+
+import java.util.List;
+
+public interface Classifier {
+    /**
+     * Trains the classifier on the given data
+     *
+     * @param dataPoints data to train with
+     * @param classLabel type of the class for the data
+     */
+    void train(List<List<Integer>> dataPoints, ClassLabel classLabel);
+
+    /**
+     * Classifies an unknown datapoint using its training
+     *
+     * @param dataPoint data to classify
+     * @return  class label
+     */
+    ClassLabel classify(Integer dataPoint);
```

Comment:
Ah okay, I just wan't sure how you would want to use classify. That is an easy change